content/pages/about.html

```diff
@@ -15,7 +15,7 @@
       <!-- Header -->
       <div id="header-wrapper">
         <div id="header" class="container">
-          <h1 id="logo"><img src="images/kinto-logo.svg" alt="Kinto"/></h1>
+          <h1 id="logo"><a href="./"><img src="images/kinto-logo.svg" alt="Kinto"/></a></h1>

```

Comment:
I think I tried but that doesn't get replaced with the actual ur, it stays that way... not very convenient